docs/configuration-validation.md

```diff
@@ -27,3 +27,5 @@ Add `configValidationMode` to your service configuration with one of the followi
 | error | Fail invoked command and output configuration errors. |
 | warn  | Output configuration errors in form of warnings.      |
 | off   | Suppress configuration errors.                        |
+
+**Note:** When not explicitly specified, `configValidationMode` defaults to `warn` before serverless@v3. From serverless@v3 onwards, it defaults to `error`.
```

Comment:
Thanks @RazCrimson - the added note is not correct - the default mode was not changed in `v3` and it's still `warn`. Do you observe different behavior?

Reply:
My bad. I mistook some other errors to be caused by this. 

Btw, Is it fine if update the PR to mention this? 
It not being present in the docs made me confused. 😅 
I initially thought that the default behavior was `off` and I had to explicitly set it to `warn` or `error`

Reply:
I think it's good to mention the default behavior, but probably not as a separate not as you propose but somewhere above where we already mention warning mode

Reply:
Is this fine?

Reply:
That sounds fine 👍 The v3 deprecation has an explanation note and should be left because some people on older versions might still hit it. Thanks for the improvement @RazCrimson 🙇